crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1368,6 +1387,21 @@ where
             - state.tx_signatures.total_votes()
     }
 
+    /// Returns the total stake that returned overload errors.
+    fn get_overloaded_stake(&self, state: &ProcessTransactionState) -> StakeUnit {
+        state
+            .errors
+            .iter()
+            .filter_map(|(error, _, stake)| {
+                if error.is_overload() {
+                    Some(stake)
+                } else {
+                    None
+                }
+            })
+            .sum()
+    }
+
```

Comment:
can we store the sum of overload stake as a variable in `state` so we don't have to re-iterate all errors every time?